types/test.ts

```diff
@@ -0,0 +1,20 @@
+import {sitemap} from 'xast-util-sitemap'
+
+sitemap([
```

Comment:
it can be handy to have a error test case as well, even a simple one like:
```suggestion

sitemap() // $ExpectError
sitemap([
```